app/views/assessments/show.html.erb

```diff
@@ -38,7 +38,6 @@
         <li> <%= link_to "Reload config file", {:action => "reload" }, {:title=> "Reload the assessment config file (provided for backward compatibility with legacy assessments)" } %></li>
         <li> <%= link_to "Manage extensions", [@course, @assessment, :extensions], { title: "Give extensions to students" } %></li>
         <li> <%= link_to "Manage submissions", [@course, @assessment, :submissions], { title: "Create, view, export, and re-autograde submissions" } %></li>
-        <li> <%= link_to "Manage assessment attachments", [@course, @assessment, :attachments], { title: "Distribute files to your students" } %></li>
```

Comment:
Since we've removed this link, there's no way to add an attachment if there isn't one there yet.